src/ws-dropdown/ws-dropdown.js

```diff
@@ -246,31 +296,47 @@ export class WSDropdown extends Component {
    * @returns {void}
    */
   open() {
-    if (this.opened || this.props.disabled) {
+    // Stop if this dropdown is already opened or close previous opened dropdown
+    if (WSDropdown.openDropdown === this || this.props.disabled) {
       return;
+    } else if (WSDropdown.openDropdown) {
+      WSDropdown.openDropdown.close();
     }
-    this.opened = true;
+
+    WSDropdown.openDropdown = this;
     this.dropdownContainer.style.height = 0;
     this.dropdownContainer.classList.add('mod-open');
     this.adjustSize(this.dropdownMenu.getHeight());
+    // Bind keyboard listener for keyboard navigation
```

Comment:
Perhaps this comment can be omitted, since it doesn't say much more than the code.